### DISCUSSION TITLE
希望能支持苹果专用密码登录

### DISCUSSION THREAD

**Discussion by yinqiong** [tryzealot/zealot#807] [Q&A] [answered]
### 描述 | Description

从一个apple 账号中 appleid.apple.com/account/manage 设置一个专用密码，然后使用专用密码登录apple 创建/获取 证书 和 描述文件，然后签名，这个流程完全自动化，如果能实现就非常棒了，fastlane 应该是未支持这样登录的

**Comment by icyleaf** [comment, marked as answer]
fastlane 的 sign 支持证书加密的，所以请仔细看官方文档:https://docs.fastlane.tools/actions/sync_code_signing/ 尤其是 `Git Storage` 部分
